etc/kayobe/kolla.yml

```diff
@@ -225,6 +225,7 @@ kolla_enable_barbican: True
 #kolla_enable_cloudkitty:
 #kolla_enable_congress:
 #kolla_enable_designate:
+kolla_enable_elasticsearch: True
```

Comment:
Will this run elasticsearch on the controller?

Reply:
Yeah - but afaik there is only a controller in the alt-1 1nv. For the production env I'll make sure it runs on the monitoring node.

Reply:
Perhaps I should map it to a monitoring group and then set that to point to the controller. It would be more similar to prod then.

Reply:
In the kolla world, yes - it could be in a monitoring group. For kayobe, controllers and monitoring are designed to be top level groups.